src/hooks/useLoadDisciplines.ts

```diff
@@ -13,8 +12,8 @@ export interface ServerResponse {
   total_documents: number
 }
 
-const getEndpoint = (parent: Discipline | null): string => {
-  if (parent?.isLeaf) {
+const getEndpoint = (parent: DisciplineType | null): string => {
+  if (parent?.total_training_sets && parent.total_training_sets > 0) {
```

Comment:
Does this really make sense?
With my requested changes in /endpoints.ts this can be reverted. If you want to have this extra check you can add it in `formatServerResponse`-function.

Reply:
See explanation above. I am open to introduce a different property which makes it clearer